com.unity.render-pipelines.core/ShaderLibrary/AreaLighting.hlsl

```diff
@@ -21,7 +21,8 @@ real3 ComputeEdgeFactor(real3 V1, real3 V2)
     if (V1oV2 < 0)
     {
         // Undo range reduction.
-        y = PI * rsqrt(saturate(1 - V1oV2 * V1oV2)) - y;
+        float f = saturate(1 - V1oV2 * V1oV2);
```

Comment:
should we do something like y = PI * rsqrt(min(Epsilon, saturate(1 - V1oV2 * V1oV2))) - y;
instead?

Reply:
We could, though that will provide an exploding value there at a specific viewing edge that I am afraid will introduce problems in bloom, however I guess we can select a decent epsilon so that doesn't happen. 

I'll do that